package.json

```diff
@@ -16,8 +16,7 @@
     "docs:lint": "sphinx-build -nW -b linkcheck docs docs/_build/",
     "docs:build": "sphinx-build -nW -b html docs docs/_build/",
     "docs:serve": "sphinx-autobuild docs docs/_build/",
-    "docs:install": "pip install -r docs/requirements.txt",
-    "lint": "conventional-changelog-lint --from=master && eslint .",
+    "lint": "conventional-changelog-lint --from=master && eslint . && npm run docs:lint",
```

Comment:
The instructions at https://github.com/apiaryio/dredd/blob/master/.github/CONTRIBUTING.md#recommended-workflow tell me to use `npm lint`, which now requires python, pip/python dependencies. I think either the docs should be updated to explain these new requirements or perhaps docs lint shouldn't be default from `lint`. When an external contributor is trying to submit a small patch they likely don't want to have to deal with setting up Python environment too.